tech-test/src/tests/SearchResults.test.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SearchResults from "../components/SearchResults";
+
+describe("SearchResults", () => {
+  const validProps = {
+    results: ["img1", "img2", "img3"]
+  };
+  
+  it("renders SearchResult component correctly", () => {
```

Comment:
Could a test be added here to check the result is as expected when no results are found? Ideally the unit tests should cover all code paths for the entity you are testing